packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/DdSessionReplayImplementationTest.kt

```diff
@@ -30,18 +37,58 @@ internal class DdSessionReplayImplementationTest {
     @Mock
     lateinit var mockPromise: Promise
 
+    @Mock
+    lateinit var mockSessionReplay: SessionReplayWrapper
+
     @BeforeEach
     fun `set up`() {
-        testedSessionReplay = DdSessionReplayImplementation()
+        testedSessionReplay = DdSessionReplayImplementation { mockSessionReplay }
     }
 
     @AfterEach
     fun `tear down`() {
     }
 
     @Test
-    fun `M do nothing W enable()`() {
+    fun `M enable session replay W enable()`(
+        forge: Forge,
+        @Forgery privacy: SessionReplayPrivacy
+    ) {
+        // Given
+        val replaySampleRate = forge.aDouble(0.0, 100.0)
+        val sessionReplayConfigCaptor = argumentCaptor<SessionReplayConfiguration>()
+
+        // When
+        testedSessionReplay.enable(replaySampleRate, privacy.toString(), mockPromise)
+
+        // Then
+        inOrder(mockSessionReplay) {
```

Comment:
there is no need to use `inOrder` here, there is only one call to `verify` here.